src/react-apps/applications/ux-editor/src/containers/EditContainer.tsx

```diff
@@ -205,6 +205,7 @@ class Edit extends React.Component<IEditContainerProps, IEditContainerState> {
       FormDesignerActionDispatchers.deleteActiveListAction();
     } else {
       FormDesignerActionDispatchers.deleteFormComponent(this.props.id);
+      FormDesignerActionDispatchers.deleteActiveListAction();
```

Comment:
Move the deleteActionListAction() out of the if-else? It should be run either way, so its redundant to have it twice.